library/nodemon.json

```
@@ -0,0 +1,6 @@
+{
+  "watch": ["src"],
+  "ext": "ts,json",
+  "ignore": ["src/**/*.spec.ts"],
+  "exec": "yarn build"
```

```suggestion
  "exec": "npm run build"
```

Maybe, so people don't need `yarn` to contribute?

---

will remove nodemon in the lib instead.`